modules/ROOT/pages/questions/question-08.adoc

```diff
@@ -0,0 +1,12 @@
+8. A re-naming happened consequent to `textpatterns` being re-cast
```

Comment:
@metricjs TINY-8312

Reply:
- textpatterns is no longer a plugin and is now part of TinyMCE core
- the previous plugin configuration to enable textpatterns is no longer required { plugins: 'textpattern' } and should be removed from the plugin configuration

more info @metricjs 
- migration - steps to migrate 5x textpattern to 6x
- configuration how to enable, disable textpattern functionality
- default configuration of textpattern

Reply:
`textpattern_patterns` is a configuration option, and I believe `text_patterns` is equivalent. Upgraders will need to rename their configuration and (as Mike said) remove `textpattern` from their plugins list. No other changes should be required.

Reply:
Yes, here's the changes:
- textpattern plugin was moved to core, so it should not be specified in the `plugins` option anymore, and it is essentially "on by default"
- `textpattern_patterns` was renamed to `text_patterns`
- you can now set `text_patterns: false` or `text_patterns: []` to turn the functionality off
- the textpattern plugin's API was removed